circleguard/enums.py

```diff
@@ -321,10 +323,14 @@ class StealDetect(Detect):
     steal_thresh: float
         If the average distance in pixels of two replays is smaller than
         this value, they are labeled cheated (stolen replays). Default 18.
+    clean_mode: :class:`~.CleanMode`
+        The options used to clean the replays before comparing them.
     """
-    def __init__(self, steal_thresh=18):
+    def __init__(self, steal_thresh=18, clean_mode=None):
```

Comment:
We should pick a sensible default for clean_mode - `CleanMode.SLOW` perhaps? Instead of making the user choose.